tests/test_functors.py

```diff
@@ -581,14 +584,159 @@ def testDipPhotometry(self):
+            (0, 0, 0, -d, np.pi),
+            # Make sure we get wrap-around to 0, 0 right
+            (2 * np.pi, 0, d, 0, np.pi / 2),
+            (2 * np.pi, 0, 0, d, 0),
+            (2 * np.pi, 0, -d, 0, -np.pi / 2),
+            (2 * np.pi, 0, 0, -d, np.pi),
+            (+0.0015, 0, 2 * np.pi - 0.05, 0, -np.pi / 2),
+            # Get another somewhat arbitrary location right [these are in rad]
+            # It's not really important to rescale d by 1/cos(dec)
+            # because we're just looking at orientation of vector
+            # but foreshadowing to the poles...
+            (0.0015, 1, 0.0015 + d, 1, np.pi / 2),
+            (0.0015, 1, 0.0015, 1 + d, 0),
+            (0.0015, 1, 0.0015 - d, 1, - np.pi / 2),
+            (0.0015, 1, 0.0015, 1 - d, np.pi),
+            # Negative dec
+            (0.0015, -1, 0.0015 + d, -1, np.pi / 2),
+            (0.0015, -1, 0.0015, -1 + d, 0),
+            (0.0015, -1, 0.0015 - d, -1, - np.pi / 2),
+            (0.0015, -1, 0.0015, -1 - d, np.pi),
+            # Make sure we get wrap-around on that right
+            (2 * np.pi + 0.0015, 1, 2 * np.pi + 0.0015 + d, 1, np.pi / 2),
+            (2 * np.pi + 0.0015, 1, 2 * np.pi + 0.0015, 1 + d, 0),
+            (2 * np.pi + 0.0015, 1, 0.0015 - d, 1, - np.pi / 2),
+            (2 * np.pi + 0.0015, 1, 0.0015, 1 - d, np.pi),
+            # Get relative wrap-around right
+            (2 * np.pi + 0.0015, 1, 0.0015 + d, 1, np.pi / 2),
+            (0.0015, 1, 2 * np.pi + 0.0015, 1 + d, 0),
+            (0.0015, 1, 2 * np.pi + 0.0015 - d, 1, - np.pi / 2),
+            (2 * np.pi + 0.0015, 1, 0.0015, 1 - d, np.pi),
+            # Get either side of RA=0 right
+            (+ d / 2, 1, 2 * np.pi - d / 2, 1, - np.pi / 2),
+            (+ d / 2, 1, - d / 2, 1, -np.pi / 2),
+            (2 * np.pi + d / 2, 1, - d / 2, 1, -np.pi / 2),
+            (-d / 2, 1, +0.0015, 1, np.pi / 2),
+            (0.0015, 1, 0.0015, 1 + d, 0),
+            (0.0015, 1, 0.0015 - d, 1, - np.pi / 2),
+            (0.0015, 1, 0.0015, 1 - d, np.pi),
+            # Try it near the poles
+            (0, np.pi / 2, 0, np.pi / 2 - d, np.pi),
+            (0, np.pi / 2 - d, 0, np.pi / 2, 0),
+            (0, -np.pi / 2, 0, -np.pi / 2 + d, 0),
+            (0, -np.pi / 2 + d, 0, -np.pi / 2, np.pi),
+        ]
+
+        df = pd.DataFrame(position_angle_test_values, columns=columns)
+
+        cd_matrix = np.array([[1, 0], [0, -1]])  # Doesn't matter because we don't use it.
+        local_wcs = LocalWcs(*cd_matrix.flatten())
+        pa = local_wcs.computePositionAngle(df["ra1"], df["dec1"], df["ra2"], df["dec2"])
+        expected = df["expected"]
+        tolerance_deg = 0.05  # degrees
+        tolerance_rad = np.deg2rad(tolerance_deg)
+
+        # Use SphereGeom to handle wrap-around separations correctly.
+        diff = []
+        for o, e in zip(pa, expected):
+            diff.append(geom.Angle(o, geom.radians).separation(geom.Angle(e, geom.radians)).asRadians())
```

Comment:
This would work better with list comprehension as

```python
diff = [geom.Angle(o, geom.radians).separation(geom.Angle(e, geom.radians)).asRadians() for o, e in zip(pa, expected)]
```

Reply:
The best way to format it with line breaks would be to run `black` on it with the line-length specified. But this repo is not fully black compliant yet, so you'd need to put this in a separate file to see what `black` would do, and that'd probably be the best solution for now. But yeah, I don't think there are any specific guidelines as to where to break otherwise.

Reply:
To be clear, I've changed this as suggested, but I would be interested in any formal style guidance.

Reply:
I have re-read the DM Python Style Guide now, and that prompted me to remove the spaces I had put around `*` and `/` operators.

Reply:
For reference, `black` here gives this:
```
        diff = [
            geom.Angle(o, geom.radians).separation(geom.Angle(e, geom.radians)).asRadians()
            for o, e in zip(pa, expected)
        ]
```

Reply:
Oh, thanks.  What would `black` do (with line length 110 characters) is excellent guidance.

Reply:
> I have re-read the DM Python Style Guide now, and that prompted me to remove the spaces I had put around * and / operators.

Ah, and I realize `black` is why I do put spaces around `*` and `/`.  But we're not `black` formatting this repo, so I should stick with the DM Python Style Guide and have no spaces.

Reply:
The suggestion to put the lines to format in a new file was an editor-agnostic one. If you use VSCode, then you could select those lines and format just those as well. I don't know which other editors have that feature.